<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Retriver</class>
 <widget class="QWidget" name="Retriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layoutGWidget">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBoxBaudrate">
     <property name="currentText">
      <string>1200</string>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelBaudrate">
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="labelCheckbit">
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxCOM"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="btnCance">
     <property name="text">
      <string>Cance</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="labelDatabit">
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBoxDatabit">
     <property name="currentText">
      <string>5</string>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="labelFlowcontrol">
     <property name="text">
      <string>流  控</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btnOpenClose">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="comboBoxFlowControl">
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hardware</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Software</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelCOM">
     <property name="text">
      <string>COM</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboBoxStopbit">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBoxCheckbit">
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelStopbit">
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>串口配置</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
